config/config_test.go

```diff
@@ -2452,6 +2491,50 @@ func TestScrapeConfigDisableCompression(t *testing.T) {
 	require.False(t, got.ScrapeConfigs[0].EnableCompression)
 }
 
+func TestScrapeConfigGlobalConvertClassicHistToNHCB(t *testing.T) {
```

Comment:
Do we need a separate test? Why not joining with the big one?

Reply:
No need for separate test. Combined with the existing one